utide/robustfit.py

```diff
@@ -103,15 +103,15 @@ def welsch(r):
     return w
```

Comment:
It can be removed, but it doesn't hurt to leave it in place until we either flesh it out or replace it.  One possibility is to use statsmodels.RLM.